lib/sharing/editor_ui.dart

```diff
@@ -206,7 +194,9 @@ abstract class EditorUi {
           modulesBaseUrl: response.modulesBaseUrl,
           addRequireJs: true,
           addFirebaseJs: shouldAddFirebaseJs,
-          destroyFrame: shouldDestroyFrame,
+          // TODO(srawlins): Determine if we need to destroy the frame when
```

Comment:
I'm curious if you've found an empirical answer to this yet. I'd wager the answer is yes, since we'd be trying to hot restart code compiled with one version of the SDKs into already running SDKs of other another version.

I'm totally willing to be controverted by reality, though. 😄

Reply:
I actually am not confident enough to say 😆 . It's a little scary.